jormungandr/Cargo.toml

```diff
@@ -63,6 +63,8 @@ jormungandr-utils = { path = "../jormungandr-utils" }
 strfmt = "0.1"
 gtmpl = "0.5.6"
 mktemp = "0.4.0"
+slog-gelf = { git = "https://github.com/jbgi/slog-gelf", rev = "2a43788a6449ed7e39b4cb2f083c5d1827d59e7b" }
```

Comment:
do we know why this was never released?

since you have forked it you could update the deps please? You are adding a lot of duplicated versions in the dependencies with this

Reply:
I can update the deps in the repo, but for my understanding, in what sense is there duplicated versions? I was thinking cargo would use the versions from Cargo.lock and not from the upstream repo...